lms/envs/common.py

```diff
@@ -2755,6 +2770,10 @@
     r'edX/org.edx.mobile',
 ]
 
+# Offset for courseware.StudentModuleHistoryExtended which is used to
+# calculate the starting primary key for the underlying table.
+STUDENTMODULEHISTORYEXTENDED_OFFSET = 10000
```

Comment:
Is it worth mentioning here that this number imposes a time limit between the migration being run and the release being deployed (if you want to avoid CSMH id overlap)?